test/common/ops/releasePets.js

```diff
@@ -0,0 +1,41 @@
+import releasePets from '../../../common/script/ops/releasePets';
+import i18n from '../../../common/script/i18n';
+import {
+  generateUser,
+} from '../../helpers/common.helper';
+import {
+  NotAuthorized,
+} from '../../../common/script/libs/errors';
+
+describe('shared.ops.releasePets', () => {
+  let user;
+  let animal = 'Wolf-Base';
+
+  beforeEach(() => {
+    user = generateUser();
+    user.items.currentPet = animal;
+    user.items.pets[animal] = 5;
+  });
+
+  it('returns an error when user balance is too low', (done) => {
+    try {
+      releasePets(user);
+    } catch (err) {
+      expect(err).to.be.an.instanceof(NotAuthorized);
+      expect(err.message).to.equal(i18n.t('notEnoughGems'));
+      done();
+    }
+  });
+
+  it('releases pets', () => {
+    user.balance = 1;
```

Comment:
These tests should be split up. This test block says "releases pets" but has assertions about the user's balance.

```js
it('releases pets', () => {
it('removes currentPet', () => {
it('increases beastMasterCount achievement', () => {
it('subtracts gems from balance', () => {
```